app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -210,31 +211,42 @@ fun SyncedTabsSectionHeader(
  */
 @Composable
 fun SyncedTabsTabItem(tabTitleText: String, url: String, onClick: () -> Unit) {
-    Column(
+    Row(
         modifier = Modifier
             .clickable(
                 onClickLabel = tabTitleText,
                 onClick = onClick
             )
             .padding(horizontal = 16.dp)
             .defaultMinSize(minHeight = 56.dp),
-        verticalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically,
     ) {
-        PrimaryText(
-            text = tabTitleText,
-            modifier = Modifier.fillMaxWidth(),
-            fontSize = 16.sp,
-            maxLines = 1
+        Favicon(
+            url = url,
+            size = 24.dp,
         )
 
-        SecondaryText(
-            text = url,
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(top = 2.dp),
-            fontSize = 12.sp,
-            maxLines = 1
-        )
+        Spacer(modifier = Modifier.width(32.dp))
+
+        Column(
+            verticalArrangement = Arrangement.Center,
+        ) {
+            PrimaryText(
+                text = tabTitleText,
+                modifier = Modifier.fillMaxWidth(),
+                fontSize = 16.sp,
+                maxLines = 1,
+            )
+
+            SecondaryText(
+                text = url,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 2.dp),
```

Comment:
I think this is okay, but makes me wonder if we should use a `Spacer`, if not when should we say to use a Spacer? An individual `Spacer` has been nice to explicitly call out spacing between 2 components.  Let me know what you decide.

Reply:
Yeah, I think this just didn't get the same treatment as other Composables in the file, so I'll go ahead and make the small refactor. I think it'll be helpful to distinguish true padding between elements (or within elements in the case of like some custom compound composable) and just whitespace (using a Spacer in this instance)